docs/CHANGELOG/changelog-0.9.x.md

```diff
@@ -1,79 +1,79 @@
+   We provide checksums in `Claudie_checksum.txt` you can verify the downloaded yaml files againts the provided checksums.
 
-3. Install claudie using `kubectl`
+3. Install Claudie using `kubectl`
 
 > We strongly recommend changing the default credentials for MongoDB, MinIO and DynamoDB before you deploy it.
 
 ```
-kubectl apply -f https://github.com/berops/claudie/releases/latest/download/claudie.yaml
+kubectl apply -f https://github.com/berops/Claudie/releases/latest/download/Claudie.yaml
 ```
 
-To further harden claudie, you may want to deploy our pre-defined network policies:
+To further harden Claudie, you may want to deploy our pre-defined network policies:
    ```bash
    # for clusters using cilium as their CNI
-   kubectl apply -f https://github.com/berops/claudie/releases/latest/download/network-policy-cilium.yaml
+   kubectl apply -f https://github.com/berops/Claudie/releases/latest/download/network-policy-cilium.yaml
    ```
    ```bash
    # other
-   kubectl apply -f https://github.com/berops/claudie/releases/latest/download/network-policy.yaml
+   kubectl apply -f https://github.com/berops/Claudie/releases/latest/download/network-policy.yaml
    ```
 
 
 ## v0.9.0
 
 
 ## What's changed
-- Support added for ubuntu 24.04 in Azure and hetzner [#1401](https://github.com/berops/claudie/pull/1401)
+- Support added for Ubuntu 24.04 in Azure and Hetzner [#1401](https://github.com/berops/Claudie/pull/1401)
 
-- Each nodepool now has its own SSH keys, a change from the previous state where all nodepools shared the same SSH keys.. [#1442](https://github.com/berops/claudie/pull/1442)
+- Each nodepool now has its own SSH keys, a change from the previous state where all nodepools shared the same SSH keys.. [#1442](https://github.com/berops/Claudie/pull/1442)
 
-- Added support for pluggable external terraform files, breaking the dependency of updated terraform files on a new claudie version. [#1460](https://github.com/berops/claudie/pull/1460)
+- Added support for pluggable external terraform files, breaking the dependency of updated terraform files on a new Claudie version. [#1460](https://github.com/berops/Claudie/pull/1460)
 
-- With the support of external terraform templates, the ability to arbitrarily change the templates used by claudie was made available to the user. As a result, claudie has implemented a rolling update of the infrastructure in case a change in the terraform templates is detected, by gradually updating the build cluster one nodepool at a time. [#1525](https://github.com/berops/claudie/pull/1525)
+- With the support of external terraform templates, the ability to arbitrarily change the templates used by Claudie was made available to the user. As a result, Claudie has implemented a rolling update of the infrastructure in case a change in the terraform templates is detected, by gradually updating the build cluster one nodepool at a time. [#1525](https://github.com/berops/Claudie/pull/1525)
```

Comment:
```suggestion
- With the support of external terraform templates, the ability to arbitrarily change the templates used by Claudie was made available to the user. As a result, Claudie has implemented a rolling update of the infrastructure in case a change in the terraform templates is detected, by gradually updating the build cluster one nodepool at a time. [#1525](https://github.com/berops/claudie/pull/1525)
```